### ISSUE TITLE
Filename

### ISSUE THREAD

**Issue by dubiousdavid** [v8/sampling-heap-profiler#9]
Not matter how I call `write` the filename is always something like:

```
{"head":{"functionName":"(root)","scriptId":0,"lineNumber":0,"columnNumber":0,"url":"","selfSize":0,"children":[]}}
```

**Comment by dubiousdavid** [comment]
The filename is never passed to this function: https://github.com/v8/sampling-heap-profiler/blob/master/src/index.ts#L77

**Comment by ofrobots** [comment]
Oops. Sorry about that. Adding a fix for it over at  https://github.com/v8/sampling-heap-profiler/pull/10

**Comment by ofrobots** [comment]
Fix released via 0.3.0. Please give this a shot.

We'd be very interested in hearing your experience debugging memory issues with this module. Let us know if this is helpful or unhelpful in your debugging.

**Comment by dubiousdavid** [comment]
Thanks! We're getting ready to deploy this in beta, and if all goes well in production.
